### PR TITLE
Merge carlos-branch to main branch

### PR DESCRIPTION
**Pull request by MrkarlosM**
We want to merge this branch with the main, in this branch we added:

- Semantic tags for each part of the page.
- Divisions `div` to make the main elements of the page.
- Different font sizes for different articles inside de webpage.
- Descriptive colors to simulate the most likely parts of the page visited by a user.

So this would help as UX designers to know which are the most-visited parts of a site and how to take advantage and make decisions about it.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,67 @@
...
+
+## Authors
+
+Please feel free to contact us in any time:
+
+👤 **Carlos Mario **
+
+- GitHub: [@githubhandle](https://github.com/MrkarlosM)
+- Twitter: [@twitterhandle](@MrkarlosM)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/carlos-mario-martinez-b1768355/)
```
Comment (descholar-ceo) [inline README.md]: You should replace @githubhandle, @twitterhandle and Linkedin texts in your links with your respective usernames you use on those sites

**Review by descholar-ceo — CHANGES_REQUESTED** [review]
# Status: Change Required :vertical_traffic_light: :warning: 
## Hello @MrkarlosM and @AbdelrhmanAmin  :handshake: :handshake: :handshake: 
:+1: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.
### :warning: However we need to make few changes to mark this project approved. See the requested change below :point_down: :point_down: :point_down: 
## Note: All of requested changes which are marked as [Required] are mandatory to be implemented in order to get this project approved, others which are marked as [Optional] are nice to have in your project to make it outstanding

# [Required Changes]
- [x] The two articles section should be centered and each column occupies the half of the width of the screen, 
Screenshot from 2020-11-03 23-07-04
:point_right: From the original [Smashing magazine website]() that place is not empty, it's filled with two columns of articles and each column occupies the half of the screen, so you should remove that empty space
- [x] Implement all changes I suggested in inline comments in your codes

# [Optional changes]
- [ ] The names of your branches and pull requests should reflect the very short summary of the work / task you have implemented on that branch / pull request. Something like `carlos-branch` is not descriptive and doesn't show what is that branch doing actually, next time try to name branches like `add-project-logic` or `implement-user-interface`, ... in that way
- [ ] Your commit messages should be in imperative mode or present tense, you should have used `Add footer items, add Footer Items and Content` instead of `Added footer items Added Footer Items and Content`, refer [to this article for more details on how to write a good && professional commit message](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8)

If anything seems to be unclear to you, don't hesitate to contact me for more clarification slack: `@Mugirase Emmanuel` | [LinkedIn](https://linkedin.com/in/mugirase-emmanuel) | [Twitter](https://twitter.com/descholar3)

Reviewed by [Mugirase Emmanuel (descholar)](https://github.com/descholar-ceo)

**Review by ebeagusamuel — CHANGES_REQUESTED** [review]
# Change Required :recycle: .
Hi team, great work you've done on this project :+1: just one change to make to get your project approved.
- [ ] Referring back to the requested change by the previous TSE, the two article section is too further apart compared to how it is on the original website. Please reduce the space between them to match the way it is positioned on the original website.

#### Once you are done making this change, please request another review.

slack(@ebeagusamuel)

**Review by descholar-ceo — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:  
## :clap: :clap: :clap: Congratulations @MrkarlosM and @AbdelrhmanAmin  , your project is approved :tada: :tada: :tada: 
You have implemented all requested changes, and your project is really great, it is marked as complete now, it doesn't need more changes, you can go ahead and merge all of the changes on base branch,

Don't forget to fill the mandatory project completion form, and see you to next project,

Happy coding :computer: :computer: :computer: 

Re-reviewed by [Mugirase Emmanuel (descholar)](https://github.com/descholar-ceo)

**Pull request merged**